scripts/install_prerequisites.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash 
```

Comment:
Hm, it's a small file, but consider putting the usual copyright headers + docstring at the top (for consistency the other scripts in scripts/)?